doc/usecases/map_timeline_event_view.md

```diff
@@ -0,0 +1,28 @@
+# Use Case: Map Timeline Event View
+
+## Summary
+
+- **Scope:** Map View
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user can see the addition/removal of elements as events using the timeline feature.
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  The user has opened the app and selected a map.
+- **Main success scenario:**
+  The user sees events represented on the scrollbar of the timeline feature.
+  Events include:
+    - addition of an element to the map
+    - deletion of an element on the map
+
+- **Alternative scenario:**
+- **Error scenario:**
+  There is an error in the timeline display or event loading functionality. 
+  In this case, the app displays an error message and allows the user to try again.
+- **Postcondition:**
+  The user successfully sees the events on the timeline scrollbar.
+- **Non-functional Constraints:**
```

Comment:
I added it for this use case as well, because it was present on the initial timeline use case.
The other use case requires about the same data then this use case.
I will also take a guess on how it will work. Upon adding/deleting an element, the changes are saved to both the local and the remote databases. If the connection is interrupted the changes can still be tracked by the local database and then synchronized upon reconnecting.
If a fresh user joins the map, all the event and timeline data is synchronized to her local database. 
How performant that is will be a part of my thesis ^^

Reply:
I understand, thanks :+1: